project/src/PlayerController.gd

```diff
@@ -30,6 +38,9 @@ func _physics_process(delta):
 	motion = motion.normalized() * speed
 	move_and_slide(motion)
 
+func is_attack_frame():
```

Comment:
I think we can add this to utils as I don't think it's the last time we will need this functionality.
something like:
```
    is_update_frame(period):
        retrun Engine.get_physics_frames() % period == 0
```